src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -50,6 +52,7 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
     }
   }
 
+
```

Comment:
```suggestion
```
Accidental newline?

Reply:
resolved in https://github.com/LokiLabs/microsoft-graph-explorer-v4/pull/16/commits/55666db4d0964581599d5f907b355fb066034d67